data/json/recipes/recipe_others.json

```diff
@@ -4666,5 +4666,17 @@
     "autolearn": true,
     "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "SAW_W", "level": 2 } ],
     "components": [ [ [ "2x4", 4 ] ], [ [ "nail", 20 ] ], [ [ "hinge", 2 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "stethoscope",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": "15 m",
+    "autolearn": true,
```

Comment:
While a child could *make* this recipe, I doubt that even many *adults* would think of it on their own. I'd like to see a book this recipe is found in, rather than having it auto-learned.

Reply:
I'll see if I can find a reasonable book(s) to contain this recipe.